src/components/Main/components/Post/PostContent.jsx

```diff
@@ -0,0 +1,24 @@
+import UserProfileCircle from "../../../Shared/UserProfileCircle"
+import NameTitleContainer from "./NameTitleContainer"
+import DeletePostButton from "../../../Shared/DeletePostButton"
+
+function PostContent({ post, URL, initials, author, setShouldGetPosts }) {
+
+
```

Comment:
Great work on this challenge, Satoki. Your code is well structured and easy to read. My only real feedback is to watch the whitespace, you have a few places with excessive empty lines

Reply:
Many thanks Nathan, appreciate the feedback 😃